File: algo-armstrong-numbers/js/armstrongNumbers.js
// How can you make this more scalable and reusable later?

exports.findArmstrongNumbers = function(arr) {
    const output = [];
    for (let num of arr) {
        const strArr = String(num).split("");
        let value = 0
        for (let y = 0; y < strArr.length; y++) {
            value += Number(strArr[y]) ** strArr.length;
        }
        if (num === value) {
            output.push(num);
        }
    }
    return output;
};

File: algo-armstrong-numbers/python/armstrong_numbers.py
# How can you make this more scalable and reusable later?
def find_armstrong_numbers(num_list):
    output = []
    for num in num_list:
        strList = [*str(num)]
        value = 0
        for elem in strList:
            value += int(elem) ** len(strList)
        if num == value:
            output.append(num)
    return output